docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,26 @@ weight: 35
 
 ## Unreleased
 
+## [v0.15.0](https://github.com/aws/eks-anywhere/releases/tag/v0.15.0)
+### Added
+- Kubernetes 1.26 support ([#5079](https://github.com/aws/eks-anywhere/issues/5079))
+- Workload clusters full lifecycle API support for Bare Metal provider ([#5237](https://github.com/aws/eks-anywhere/issues/5237))
+- Workload clusters full lifecycle API support for Nutanix provider
```

Comment:
nit: we can add this for Nutanix https://github.com/aws/eks-anywhere/pull/5190

Reply:
So, we can add pull requests instead of issues?

Reply:
yes we have done this previously for features we didn't create issues for but had a significant PR for it that had some description.